content/1.2-rc/docs/serverless-workflows/troubleshooting.md

```diff
@@ -116,6 +117,12 @@ In this configuration:
 
 By following these steps, the workflow will have the necessary credentials to access PostgreSQL and will correctly reference the service in a different namespace.
 
+## HTTP Errors
+Many workflow operations are REST requests to HTTP servers. If an HTTP error occurs then the workflow will fail and the HTTP code and message will be displayed. Please use [HTTP codes documentation](https://developer.mozilla.org/docs/Web/HTTP/Status) for understanding the meaning of such errors.  
```

Comment:
Instead of HTTP servers I would say REST endpoints

Reply:
but it is an http server and we are talking about HTTP status codes. no point trying to hide it

Reply:
fixed